src/app/pages/ajustes/lista-movimientos/lista-movimientos.component.html

```diff
@@ -3,12 +3,12 @@
     <nb-card-header>
       Registrar Ajuste
     </nb-card-header>
-    <nb-card-body [nbSpinner]="!mostrar">
+    <nb-card-body [nbSpinner]="false">
       <p class="h4 pb-4" *ngIf="actaSeleccionada === ''">{{'GLOBAL.mensaje_registrar_entrada' | translate}}</p>
 
       <ng2-smart-table [settings]="settingsListaActas" [source]="ActasAsociadas"
         (custom)="CargarMovimientosAsociados($event)" *ngIf="actaSeleccionada === ''"></ng2-smart-table>
-      <nb-card *ngIf="actaSeleccionada !== ''" [nbSpinner]="!mostrarActa || !mostrarMovimientos || anulando">
+      <nb-card *ngIf="actaSeleccionada !== ''" [nbSpinner]="false">
```

Comment:
Si el valor no es dinámico, se puede omitir el data binding
```suggestion
      <nb-card *ngIf="actaSeleccionada !== ''" nbSpinner="false">
```

O mejor aún, se puede quitar esta directiva
```suggestion
      <nb-card *ngIf="actaSeleccionada !== ''">
```